### DISCUSSION TITLE
Indexing Error: "Cannot get page number" build number 35304

### DISCUSSION THREAD

**Discussion by busbina** [ravendb/ravendb#12855] [Q&A] [answered]
I am getting the following indexing error in Server Build #35304

> Cannot get page number 8673772830359290880 because number of allocated pages is 47448080

Not sure what this means and cant find any other conversation about it.

**Comment by ayende** [comment, marked as answer]
This is a version that is not longer supported. 
Please migrate to version 5.2

**Reply by busbina** [reply]
Thanks, forgot that was out of support. I exported and smuggled the DB back in and that fixed the problem anyways.
